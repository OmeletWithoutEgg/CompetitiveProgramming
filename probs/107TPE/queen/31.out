-454593.26 -66537238.33
