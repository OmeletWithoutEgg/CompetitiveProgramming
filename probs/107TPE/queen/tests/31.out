-28358985.00 91898949.00
